### DISCUSSION TITLE
添加 BDD100K 语义分割数据集

### DISCUSSION THREAD

**Discussion by OpenMMLab-Assistant-004** [open-mmlab/OpenMMLabCamp#366] [MMSegmentation]
# 添加 BDD100K 语义分割数据集 

[https://github.com/open-mmlab/mmsegmentation/issues/2808](https://github.com/open-mmlab/mmsegmentation/issues/2808) ，将代码加入到 project 中，提供一个简单的配置文件，能够用这个数据集启动训练。

## Task Information

**Repository**：MMSegmentation

**Difficulty**：Easy

**Points**：20

**Estimated Time Consumption**：7 days

## Task Status

**Claimant's Github ID**：CastleDream

**Claim Time**：2023年06月20日

**PR Link**：https://github.com/open-mmlab/mmsegmentation/pull/3158

## Points to Note

To claim the task, please reply `Claim the mission` in the comment section. After claiming the task, please add our assistant bot called MeowMewo on WeChat at: `openmmlabwx`.

After completing the task, please submit a PR with the title starting with [MMSIG]. After submission, please reply with your PR link in the comments section of this Discussion. We will review your work as soon as possible.

在评论区回复 `Claim the mission` 领取任务，领取任务后请及时添加小助手喵喵微信: `openmmlabwx`

完成任务后提交相关链接（如是PR，标题请以[MMSIG]开头）回复到本Disucssion评论区，我们会第一时间审核您的贡献。

**Comment by CastleDream** [comment]
我的pr已经合并了我才看见这个新版的任务领取规则，好像没法改PR的名字了
https://github.com/open-mmlab/mmsegmentation/pull/3158

**Reply by crazysteeaam** [reply]
good job~
